src/components/build/playPreview/PlayPreviewMenu.tsx

```diff
@@ -54,7 +54,7 @@ class PlayPreviewMenu extends Component<PreviewMenuProps, PreviewMenuState> {
         <PageHeader
           searchPlaceholder="Search Subjects, Topics, Titles &amp; more"
           search={() => {}}
-          searching={(v) => {}}
+          searching={(v: string) => {}}
```

Comment:
@william-gooch  v is incoming value. Type should be defined in the PageHeader component. there is no need to define it here

Reply:
I understand this - it was working before. However using the connector in PageHeader causes eslint to throw an error in every case where this appears. I tried for a while to figure out why but it won't compile without it, even though the type is defined in the props interface. Any idea why this might be happening?